concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -0,0 +1,331 @@
+Desrochers-2010-330-178	178	libro	book	BOOK	963	Noun	6.67647058823529	5	1088	6.84
+Desrochers-2010-330-179	179	línea	line	LINE	1048	Noun	5.99019607843137	5	488	6.15
+Desrochers-2010-330-180	180	llama	flame	FLAME	889	Noun	5.05882352941176	5	631	4.95
+Desrochers-2010-330-181	181	llave	key	KEY	1472	Noun	6.63725490196079	5	128	6.71
+Desrochers-2010-330-182	182	mañana	tomorrow	TOMORROW	1329	Noun	6.69607843137255	6	1429	7
+Desrochers-2010-330-183	183	mancha	stain			Noun	6.05882352941176	6	91	6.56
+Desrochers-2010-330-184	184	mano	hand	HAND	1277	Noun	6.57843137254902	4	2168	6.53
+Desrochers-2010-330-185	185	mantel	tablecloth			Noun	5.84313725490196	6	33	6.27
+Desrochers-2010-330-186	186	marciano	martian			Noun	4.34313725490196	8	8	
+Desrochers-2010-330-187	187	miedo	fear	FEAR (FRIGHT)	781	Noun	6.16666666666667	5	839	6.19
+Desrochers-2010-330-188	188	moneda	currency			Noun	6.57843137254902	6	110	6.31
+Desrochers-2010-330-189	189	monstruo	monster			Noun	5.14705882352941	8	117	6.06
+Desrochers-2010-330-190	190	montaña	mountain	MOUNTAIN	639	Noun	6.08823529411765	7	193	6.38
+Desrochers-2010-330-191	191	mosca	fly	FLY (INSECT)	1504	Noun	6.01960784313726	5	60	5.94
+Desrochers-2010-330-192	192	mueble	furniture			Noun	6.17647058823529	6	42	6.27
+Desrochers-2010-330-193	193	mundo	world	WORLD	965	Noun	6.44117647058824	5	3783	6.44
+Desrochers-2010-330-194	194	nieve	snow	SNOW	784	Noun	5.82352941176471	5	220	5.41
+Desrochers-2010-330-195	195	nuez	nut	NUT	191	Noun	5.42156862745098	4	24	5.17
+Desrochers-2010-330-196	196	ola	wave	WAVE	978	Noun	5.5	3	62	6.76
+Desrochers-2010-330-197	197	página	page			Noun	6.52941176470588	6	178	6.64
+Desrochers-2010-330-198	198	pájaro	bird	BIRD	937	Noun	6.29411764705882	6	116	6.5
+Desrochers-2010-330-199	199	papel	paper	PAPER	648	Noun	6.66666666666667	5	1028	6.66
+Desrochers-2010-330-200	200	paso	step	STEP	1158	Noun	5.88235294117647	4	1331	6.24
+Desrochers-2010-330-201	201	pata	paw	PAW	2965	Noun	5.92156862745098	4	78	6.45
+Desrochers-2010-330-202	202	patata	potato	POTATO	593	Noun	6.51960784313726	6	33	6.75
+Desrochers-2010-330-203	203	pelota	ball	BALL	1845	Noun	6.40196078431373	6	103	6.49
+Desrochers-2010-330-204	204	perra	bitch			Noun	6.22	5	60	6.47
+Desrochers-2010-330-205	205	persona	person	PERSON	683	Noun	6.50980392156863	7	962	7
+Desrochers-2010-330-206	206	pizza	pizza			Noun	6.24509803921569	5	5	
+Desrochers-2010-330-207	207	plato	plate	PLATE	480	Noun	6.6078431372549	5	173	
+Desrochers-2010-330-208	208	pluma	pen	PEN	794	Noun	5.48039215686275	5	89	5.76
+Desrochers-2010-330-209	209	puerta	door	DOOR	1567	Noun	6.57843137254902	6	1557	6.71
+Desrochers-2010-330-210	210	punta	tip	TIP (OF OBJECT)	2992	Noun	5.89114925588637	5	232	5.69
+Desrochers-2010-330-211	211	puro	pure			Noun	5.42156862745098	4	236	5.33
+Desrochers-2010-330-212	212	rabia	rabies	RABIES	2206	Noun	5.51960784313726	5	128	6.18
+Desrochers-2010-330-213	213	rata	rat	RAT	1490	Noun	5.79411764705882	4	70	6.3
+Desrochers-2010-330-214	214	rayo	ray			Noun	5.15686274509804	4	111	5.86
```

Comment:
rayo is also "blitz"

Reply:
The problem is it is ambiguous, it is "lichtstrahl" and "blitz". But one could link to "lightning" for the time being.

Reply:
It is indeed ambiguous, but I agree it is probably "lightning".

Reply:
When used in the sense of "ray/Lichtstrahl", my impression is that people would qualify it if there is no context (e.g. "rayo de sol" - "sun ray")